script/01_PoolManager.s.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import "forge-std/console2.sol";
+import "forge-std/Script.sol";
+import {PoolManager} from "../src/PoolManager.sol";
+import {IPoolManager} from "../src/interfaces/IPoolManager.sol";
+
+import "forge-std/console2.sol";
+
+contract DeployPoolManager is Script {
+    function setUp() public {}
+
+    function run(uint256 controllerGasLimit) public returns (IPoolManager manager) {
+        vm.startBroadcast();
+
+        manager = new PoolManager(controllerGasLimit);
```

Comment:
It'd be cool to have a universal vanity address across chains using `CREATE2` or `CREATE3` deployment.

Reply:
Thats the plan :) this is just a temporary script for our testnet deployments